data_utils.py

```diff
@@ -0,0 +1,128 @@
+from constants import FEATURES
```

Comment:
Do relative import. For that purpose, place your Python files within a single package and then do `from .constants import FEATURES`

Reply:
moved .py files into wcst_encode/, moved .ipynb files into notebooks/